packages/igx-templates/igx-ts/bullet-graph/default/files/src/app/__path__/__filePrefix__.component.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { <%=ClassName%>Component } from './<%=filePrefix%>.component';
 import { NoopAnimationsModule } from '@angular/platform-browser/animations';
-import { IgxBulletGraphModule } from 'igniteui-angular-gauges/ES5/igx-bullet-graph-module';
+import { IgxBulletGraphModule } from 'igniteui-angular-gauges';
```

Comment:
Side question - do those sub path imports still work after the update?

Reply:
Nope. We can maybe make a migration for this? In our CLI or the main package. We still can't cover the case where someone strictly runs `ng update igniteui-angular-gauges` though